docs/SDK/Explanation/authentication/sessionSigs/authMethods/webAuthn.md

```diff
@@ -0,0 +1,89 @@
+
+```javascript
+// Set up LitAuthClient
+const litAuthClient = new LitAuthClient({
+  litRelayConfig: {
+     // Request a Lit Relay Server API key here: https://forms.gle/RNZYtGYTY9BcD9MEA
+    relayApiKey: '<Your Lit Relay Server API Key>',
+  },
+});
+
+// Initialize WebAuthn provider
+litAuthClient.initProvider(ProviderType.WebAuthn);
+
+async function registerWithWebAuthn() {
+  const provider = litAuthClient.getProvider(ProviderType.WebAuthn);
+  // Register new WebAuthn credential
+  const options = await provider.register();
+
+  // Verify registration and mint PKP through relay server
+  const txHash = await provider.verifyAndMintPKPThroughRelayer(options);
+  const response = await provider.relay.pollRequestUntilTerminalState(
+    txHash
+  );
+  // Return public key of newly minted PKP
+  return response.pkpPublicKey;
+}
+```
+
+:::note
+
+The Lit Relay Server enables you to mint PKPs without worrying about gas fees. You can also use your own relay server or mint PKPs directly using Lit's contracts.
+
+If you are using Lit Relay Server, you will need to request an API key [here](https://forms.gle/RNZYtGYTY9BcD9MEA).
+
+:::
+
+## Authenticating a Credential
+
+Authentication is similar to logging in with an existing account. During the authentication process, the user is prompted to sign a challenge. The signed challenge is then sent to the Lit nodes, which verify the signature and generates a threshold signature of an `AuthSig` for the associated PKP.
+
+```javascript
+async function authenticateWithWebAuthn() {
+  const provider = litAuthClient.getProvider(ProviderType.WebAuthn);
+  const authMethod = await provider.authenticate();
+  return authMethod;
+}
+```
+
+The `authenticate` method returns an `AuthMethod` object containing the authentication data. You can use the authentication data to mint or fetch PKPs associated with the verified WebAuthn credential.
+
+## Generating `SessionSigs`
+
+After successfully authenticating with a WebAuthn credential, you can generate `SessionSigs` using the provider's `getSessionSigs` method. The `getSessionSigs` method takes in an `AuthMethod` object, a PKP public key, and other session-specific arguments such as `resourceAbilityRequests` and returns a `SessionSig` object.
+
+```javascript
+// Get session signatures for the given PKP public key and auth method
+const sessionSigs = await provider.getSessionSigs({
+  authMethod: '<AuthMethod object returned from authenticate()>',
+  sessionSigsParams: {
+    chain: 'ethereum',
+    resourceAbilityRequests: [
+      resource: litResource,
+      ability: LitAbility.AccessControlConditionDecryption
+    ],
+  },
+});
+```
+
+## Technical Details
+
+While the registration step involves minting a PKP via the Lit Relay Server (or, optionally, you can send a transaction to the contract yourself), we have implemented an authentication scheme that involves sending signed challenges to the decentralized Lit Network instead.
+
```

Comment:
it might be nice to list here what data is needed to send to the contract when registering webauthn credentials in a bulleted list format. We break it down a bit below, but its a little wordy. maybe being explicit with example data so people can better understand what is needed if not minting through the relayer.

Reply:
Will do but also, it's a bit hard to understand anyways since our contracts are still closed source and, unless their look on caldera + we publish & verify on there, it's pretty hard to get in plaintext.